src/pymodaq_utils/utils.py

```diff
@@ -29,13 +22,6 @@
 PackageNotFoundError = metadata.PackageNotFoundError  # for use elsewhere
 
 
-if version_mod.parse(platform.python_version()) >= version_mod.parse('3.9'):
```

Comment:
why did you remove this? This is used elsewhere as an import 
maybe just add the comment:   # for use elsewhere